src/NexusMods.App.UI/Resources/Language.pl.resx

```diff
@@ -473,6 +473,18 @@
     <data name="LoadoutGridView__Remove" xml:space="preserve">
         <value>Usuń</value>
     </data>
+    <data name="FileOriginsPage_GetMods" xml:space="preserve">
+        <value>Pobierz mody:</value>
```

Comment:
I suggest changing `Pobierz` (Download) to `Uzyskaj` (Get), because `Pobierz` (Download) does not match to `Z dyksu` (From Drive).

```suggestion
          <value>Uzyskaj mody:</value>
```